### PR TITLE
Added option to hide Discord Shop button

### LINKED ISSUE #15: new "shop" button
image

new shop button cant be disabled with this plugin yet

### PR DESCRIPTION
**Pull request by LancersBucket**
- Added ability to disable Discord Shop button and added config setting.
- Closes #15

### CODE REVIEW

**Comment by 7MXek** [comment]
I'm eagerly awaiting this merge request to be approved. @QbDesu, is this plugin still being maintained?

**Comment by deepspaceaxolotl** [comment]
I would also like to know when this will be approved.

**Comment by MagmaSlime123** [comment]
If this plugin is dead, please remove it. If it isn't, please merge this.

**Comment by 7MXek** [comment]
> If this plugin is dead, please remove it. If it isn't, please merge this.

There has been absolutely no activity for over a year. I think we're going to need someone to fork the repository.

**Comment by whataboutism-alos** [comment]
please 🙏 
just edited the contents into my own file, woohoo
it works great

File: RemoveChatButtons.plugin.js [hunk 1]
```diff
@@ -186,6 +186,14 @@ module.exports = (() => {
...
                         value: true,
                     },
+                    {
+                        type: 'switch',
+                        id: 'discordShopTab',
+                        name: 'Remove Discord\'s Shop Tab',
+                        note: 'Removes the Discord Shop tab button from the DM list.',
+                        value: false,
+
+                    }
```
Comment (Ecconia) [inline RemoveChatButtons.plugin.js]: There should be a trailing comma here to maintain the file style (and keep future additions clean).

**Review by Ecconia — CHANGES_REQUESTED** [review]
Dead repo - random guy reviews :P
(Thanks for this PR, swiftly applied it locally).

**Comment by MagmaSlime123** [comment]
> Dead repo - random guy reviews :P (Thanks for this PR, swiftly applied it locally).

Then you should fork and continue this yourself.

**Pull request closed without merging**